### ISSUE TITLE
Fix accessibility issues from Pa11y audit: elements and attributes

### ISSUE THREAD

**Issue by brentryanjohnson** [ONRR/doi-extractives-data#4069]
• Error: The html element should have a lang or xml:lang attribute which describes the language of the document.
├── WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
├── html
└── <me...

• Error: This select element does not have a name available to an accessibility API. Valid names are: label element, title attribute, aria-label attribute, aria-labelledby attribute.
├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
├── #data-summary > section:nth-child(3) > div:nth-child(4) > div:nth-child(2) > div > select
└── C...

• Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
├── #data-summary > section:nth-child(3) > div:nth-child(4) > div:nth-child(2) > div > select
└── C...

• Error: This select element does not have a name available to an accessibility API. Valid names are: label element, title attribute, aria-label attribute, aria-labelledby attribute.
├── WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.Select.Name
├── #key-stats-production > div:nth-child(2) > div:nth-child(2) > div > select
└── C...

• Error: This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.
├── WCAG2AA.Principle1.Guideline1_3.1_3_1.F68
├── #key-stats-production > div:nth-child(2) > div:nth-child(2) > div > select
└── C...

**Comment by master12** [comment]
There is a "lang" attribute on all of our generated pages. Not sure what page that was referring to. @brentryanjohnson

**Comment by brentryanjohnson** [comment]
When I created this issue, the `lang` attribute was missing. I added the `lang` attribute to `DefaultLayout` [in this PR](https://github.com/ONRR/doi-extractives-data/pull/4113/files).

**Comment by master12** [comment]
Ok, decided not to fix stuff on homepage that I know Mojo is working on. I ran an audit using axe chrome extension. Fixed the Search and Glossary components. Also ran it on about and howitworks pages and fixed a few violations there. I am creating a PR and we can continue working on accessibility  with new tickets. I would suggest we create tickets for various areas of the site to review.
